frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/index.tsx

```diff
@@ -28,11 +23,10 @@ interface Props {
   values: SelectCategoryValueView[];
 }
 
-const ADDITIONAL_MENU_WIDTH = 8;
+const ADDITIONAL_MENU_WIDTH = 24;
```

Comment:
Increase in menu width to accommodate selected front weight of `600`.

Reply:
nit: should we add this as a comment in code? Feels like a good context to have!